domains/khushpatil.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "my personal homepage",
+  "repo": "https://github.com/AssassinK786",
+  "owner": {
+    "username": "AssassinK786",
+    "email": "khushpatil01@gmail.com"
+  },
+  "record": {
+    "CNAME": "AssassinK786.github.io/AssassinK786"
```

```suggestion
    "CNAME": "AssassinK786.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).